addons/kotsadm/1.86.0/install.sh

```diff
@@ -326,7 +326,7 @@ function kotsadm_confg_configmap() {
     local dst="$1"
 
     if ! kubernetes_resource_exists default configmap kotsadm-confg; then
-        kubectl -n default create configmap kotsadm-confg
+        kubectl -n default create configmap kotsadm-confg --save-config
```

Comment:
Can you make this change [here](https://github.com/replicatedhq/kots/blob/2629831e12905ca5cabb306f1b0f5adb5a7d34c2/deploy/kurl/kotsadm/template/base/install.sh#L347) rather than in the kurl project?

Reply:
Thanks, I had completely forgotten about it.